BuildScripts/Build-Ares.ps1

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env pwsh
+#Requires -Version 7.2
+
+[CmdletBinding()]
+param (
+  [Parameter()]
+  [string]
+  $Configuration = 'Release',
+  [Parameter()]
+  [Switch]
+  [bool]
+  $SkipMoveLibraries
+)
+
+# Imports
+. (Join-Path $PSScriptRoot "lib" "Enums.ps1")
+. (Join-Path $PSScriptRoot "lib" "BuildTools.ps1")
+
+Build-Ares $Configuration -SkipMoveLibraries:$SkipMoveLibraries
```

Comment:
IMO top-level build script files should be few lines for just calling stuff with proper parameters, except the build all one.